build.gradle

```diff
@@ -123,19 +123,32 @@ jacocoTestReport {
     }
 }
 
-tasks.withType(FindBugs) {
-    reports {
-        xml.enabled false
-        html.enabled true
-    }
+spotbugs {
+  toolVersion = '4.0.0-beta4'
+  ignoreFailures = true
+}
+
+// To generate an HTML report instead of XML <-- This does not work at the moment for unknown reasons. Maybe with the next release of spotbugs. Until then only XML output is possible.
```

Comment:
You mean until then "only html" is possible but it's obvious from the code below so no urgent need to fix this and go down the CI hell ;-)

Reply:
No. I actually mean only XML. But the comment is so old, that we already have one of the next releases.